### PR TITLE
wscript: Fix _check_mac_osx_version() on modern MacOS systems

### PR DESCRIPTION
**Pull request by jhunkeler**
If there's no PATCH version available then handle it.

Close #570

### CODE REVIEW

**Comment by jhunkeler** [comment]
#570

File: wscript [hunk 1]
```diff
@@ -317,7 +317,13 @@ def _check_mac_osx_version(floor_version):
...
 
     # Extract the integer values between the '.'s
-    osx_version_major, osx_version_minor, osx_version_patch = tuple(int(x) for x in s.strip().split('.'))
+    osx_version_data = tuple(int(x) for x in s.strip().split('.'))
+    osx_version_major = osx_version_data[0]
+    osx_version_minor = osx_version_data[1]
+    osx_version_patch = 0
+
+    if len(osx_version_data) > 2:
+        osx_version_patch = osx_version_data[2]
```
Comment (pllim) [inline wscript:326]:
```suggestion
    if len(osx_version_data) > 2:
        osx_version_patch = osx_version_data[2]
    else:
        osx_version_patch = 0
```

**Comment by pllim** [comment]
I dunno why Jenkins has a ❌ . I cannot access the log.

**Comment by mdlpstsci** [comment]
The problem is accessing 'ssb.stsci.edu' as there is a certificate issue.  This has been a problem for many folks and there is a ticket out for ITSD to address the issue.

File: wscript (hunk 1 above)
Comment (mdlpstsci) [reply wscript:326]: Both solutions look fine though I prefer the JH solution as the variable has a value by default.

**Review by mdlpstsci — APPROVED** [review]

**Comment by pllim** [comment]
Feel free to ignore my suggestion then. Thanks, everyone!

**Pull request merged**